packages/core/contracts/cross-chain-oracle/GovernorSpoke.sol

```diff
@@ -37,15 +48,27 @@ contract GovernorSpoke is Lockable, ChildMessengerConsumerInterface {
      */
     function processMessageFromParent(bytes memory data) public override nonReentrant() onlyMessenger() {
         (address to, bytes memory inputData) = abi.decode(data, (address, bytes));
-        // TODO: Consider calling this via <address>.call(): https://docs.soliditylang.org/en/v0.8.10/units-and-global-variables.html?highlight=low%20level%20call#members-of-address-types
-        // to avoid inline assembly.
 
         require(_executeCall(to, inputData), "execute call failed");
-        emit ExecutedGovernanceTransaction(to, inputData);
+        emit ExecutedGovernanceTransaction(to, data);
```

Comment:
Since "to" is a part of "data" I think you should either emit "to" and "inputData" (the data you call "to" with) or just data. I prefer leaving this the way it is

Reply:
agreed! reverted